### PR TITLE
Adding link to blog

### PR DESCRIPTION
**Pull request by SnehaGunda**

### CODE REVIEW

**Comment by openpublishbuild** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/spring-framework/configure-spring-boot-starter-java-app-with-cosmos-db.md](https://github.com/SnehaGunda/azure-docs-sdk-java/blob/patch-1/docs-ref-conceptual/spring-framework/configure-spring-boot-starter-java-app-with-cosmos-db.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C4%5C19%5C5de83821-f44b-0e93-473e-c19393f102fe%5CPullRequest%5C201804192124255287-426%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=YrTyQ1JNPnZQNldnBoMkXRim3vfhw%2BYBGhtRVsP9a4E%3D&st=2018-04-19T21%3A46%3A25Z&se=2018-05-20T21%3A51%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by SnehaGunda** [comment]
#sign-off

File: docs-ref-conceptual/spring-framework/configure-spring-boot-starter-java-app-with-cosmos-db.md [hunk 1]
```diff
@@ -294,6 +294,8 @@ For more information about using Azure Cosmos DB and Java, see the following art
 
 * [Azure Cosmos DB: Create a document database using Java and the Azure portal][Build a SQL API app with Java]
```
Comment (brunoborges) [inline docs-ref-conceptual/spring-framework/configure-spring-boot-starter-java-app-with-cosmos-db.md]: Could you please also provide the correct link to the item above, in this PR? 

https://docs.microsoft.com/en-us/azure/cosmos-db/create-sql-api-java

**Comment by SnehaGunda** [comment]
@brunoborges Its already linked to the right doc. There are two approached for linking, adding an inline link or referring to the link with its shortcut name. If you see raw markdown content of this file, there is a list of URIs at the end of doc and this URI is referred while linking. 



[Azure Cosmos DB Documentation]: /azure/cosmos-db/
[Azure for Java Developers]: https://docs.microsoft.com/java/azure/
[Build a SQL API app with Java]: https://docs.microsoft.com/azure/cosmos-db/create-sql-api-java 
[free Azure account]: https://azure.microsoft.com/pricing/free-trial/
[Java Tools for Visual Studio Team Services]: https://java.visualstudio.com/
[MSDN subscriber benefits]: https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/
[Spring Boot]: http://projects.spring.io/spring-boot/
[Spring Initializr]: https://start.spring.io/
[Spring Framework]: https://spring.io/

**Comment by SnehaGunda** [comment]
#sign-off

**Pull request merged**